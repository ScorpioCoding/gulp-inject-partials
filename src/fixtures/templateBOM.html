<!DOCTYPE html>
<html>
<head>
	<title>gulp-inject</title>
</head>
<body>
	<!-- partial:partialBOM.html -->
	<!-- partial -->
</body>
</html>