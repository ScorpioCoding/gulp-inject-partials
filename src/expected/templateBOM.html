<!DOCTYPE html>
<html>
<head>
	<title>gulp-inject</title>
</head>
<body>
	<!-- partial:partialBOM.html -->
	<div>In partial with BOM</div>
	<!-- partial -->
</body>
</html>